crates/ruff_python_ast/src/nodes.rs

```diff
@@ -2727,125 +2727,13 @@ pub enum ExprContext {
     Store,
     Del,
 }
-impl ExprContext {
-    #[inline]
-    pub const fn load(&self) -> Option<ExprContextLoad> {
-        match self {
-            ExprContext::Load => Some(ExprContextLoad),
-            _ => None,
-        }
-    }
-
-    #[inline]
-    pub const fn store(&self) -> Option<ExprContextStore> {
-        match self {
-            ExprContext::Store => Some(ExprContextStore),
-            _ => None,
-        }
-    }
-
-    #[inline]
-    pub const fn del(&self) -> Option<ExprContextDel> {
-        match self {
-            ExprContext::Del => Some(ExprContextDel),
-            _ => None,
-        }
-    }
-}
-
-pub struct ExprContextLoad;
```

Comment:
Hum, the `ExprContext*` star types were interesting... Why did they exist when `ExprContext::Load`, `::Store` etc. exits too. Anyway, it makes sense to remove them

Reply:
Yeah, I'm not sure either, maybe to convert to a Python class https://github.com/RustPython/Parser/blob/9ce55aefdeb35e2f706ce0b02d5a2dfe6295fc57/ast-pyo3/src/gen/to_py_ast.rs#L987-L997 ?